src/helpDB.jl

```diff
@@ -0,0 +1,296 @@
+# This file is part of the TaylorSeries.jl Julia package, MIT license
+#
+#`docstrings` for the Taylor1.jl and TaylorN.jl scripts
+#
+
+
+                                        ### Exclusive Operations ###
+
+
+
+## Constructors ##
+@doc """
+    Taylor1{T<:Number} <: Number
+
+DataType for polynomial expansions in one independent variable.
+
+**Fields:**
+
+- `coeffs :: Array{T,1}` Expansion coefficients; the \$i\$-th
+component is the coefficient of degree \$i-1\$ of the expansion.
+- `order  :: Int64` Maximum order (degree) of the polynomial.
+"""->
+Taylor1
+
+@doc """
+    taylor1_variable(T, [order=1])
+    taylor1_variable([order=1])
+
+Short-cut to define the independent variable as a `Taylor1` polynomial of
+given `order`. If `T::Type` is ommitted, `Float64` is assumend.
+"""-> 
+taylor1_variable
+
+## get_coeff ##
+@doc """
+    get_coeff(a, n)
+
+Return the coefficient of order `n::Int` of a `a::Taylor1` polynomial.
+""" ->
+get_coeff
+
+## Differentiating ##
+doc"""
+    diffTaylor(a)
```

Comment:
OK, agreed. What is the difference between `diffTaylor` and `deriv` then? Can they somehow be combined into `derivative`?